pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "djangorestframework"
+readme = "README.md"
+authors = [{name = "Tom Christie", email = "tom@tomchristie.com"}]
+license = {text = "BSD"}
+description = "Web APIs for Django, made easy."
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Web Environment",
+    "Framework :: Django",
+    "Framework :: Django :: 3.0",
+    "Framework :: Django :: 3.1",
+    "Framework :: Django :: 3.2",
+    "Framework :: Django :: 4.0",
+    "Framework :: Django :: 4.1",
+    "Framework :: Django :: 4.2",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Internet :: WWW/HTTP",
+]
+requires-python = ">=3.7"
+dependencies = [
+    "django>=3.0",
+    'backports.zoneinfo;python_version<"3.9"',
+]
+dynamic = ["version"]
+
+[tool.setuptools.dynamic]
+version = {attr = "rest_framework.__version__"}
+
+[project.urls]
+Homepage = "https://www.django-rest-framework.org/"
+Funding = "https://fund.django-rest-framework.org/topics/funding/"
+Source = "https://github.com/encode/django-rest-framework"
+Changelog = "https://www.django-rest-framework.org/community/release-notes/"
+
+[tool.setuptools]
+include-package-data = true
```

Comment:
Since `setuptools>=61.0.0`, this is the default when using `pyproject.toml`

Reply:
TILS